LINUX/packaging/netmap.spec

```diff
@@ -0,0 +1,54 @@
+Name:          netmap
+Version:       %{version}
+Release:       %{buildnumber}%{?dist}
+Summary:       netmap for Network Monitor
+Group:         Development/Tools
+License:       Dual BSD/GPL
+BuildRequires: kernel-devel = 2.6.32-573.7.1.el6
+Requires:      kernel = 2.6.32-573.7.1.el6
+ExclusiveArch: x86_64
+
+%description
+
+%prep
+cd ~/rpmbuild/BUILD
+/bin/rm -rf %{name}
+/bin/mkdir %{name}
+cd %{name}
+/bin/tar xzf ~/rpmbuild/SOURCES/%{name}-%{version}.%{buildnumber}.tar.gz
+if [ $? -ne 0 ]; then
+   exit $?
+fi
+
+%build
+# SKIP_BUILD_RPATH, CMAKE_SKIP_BUILD_RPATH, 
+cd %{name}/LINUX/
+PATH=/usr/local/probe/bin:$PATH
+./configure --no-drivers
+make
+
+%install
+/bin/mkdir -p $RPM_BUILD_ROOT/lib/modules/2.6.32-573.7.1.el6.x86_64/extra/
+cd ~/rpmbuild/BUILD/%{name}/
+/bin/cp LINUX/netmap.ko $RPM_BUILD_ROOT/lib/modules/2.6.32-573.7.1.el6.x86_64/extra/
```

Comment:
If you want netmap main repository to take this in, then it would make sense that this version should be configurable. 
Example: On CentOS 7 this would be a different kernel version. 

Please make the kernel version configurable through the buildRpm.sh script. Good  comments there to help other companies/users understand why would probably not be a bad idea either.